client/packages/system/src/Program/Components/ProgramSearchModal/ProgramSearchModal.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC } from 'react';
+import {
+  createQueryParamsStore,
+  ListSearch,
+  QueryParamsProvider,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { useProgram, ProgramDocumentFragment } from '../../api';
+import { filterByType, ProgramSearchProps } from '../../utils';
+import { getProgramOptionRenderer } from './ProgramOptionRenderer';
+// import { usePatient } from '@openmsupply-client/system/src/Patient/api';
+
+const ProgramSearchComponent: FC<ProgramSearchProps> = ({
+  disabledPrograms = [],
+  open,
+  onClose,
+  onChange,
+}) => {
+  const { data, isLoading } = useProgram.document.list();
+  // const patientId = usePatient.utils.id();
+  const t = useTranslation('app');
+  const ProgramOptionRenderer = getProgramOptionRenderer();
+
+  return (
+    <ListSearch<ProgramDocumentFragment>
+      loading={isLoading}
+      open={open}
+      options={data?.nodes ?? []}
+      onClose={onClose}
+      title={t('programs')}
+      renderOption={ProgramOptionRenderer}
+      getOptionLabel={(option: ProgramDocumentFragment) => option.name ?? ''}
+      filterOptions={filterByType}
+      onChange={(
+        _,
+        name: ProgramDocumentFragment | ProgramDocumentFragment[] | null
+      ) => {
+        if (name && !(name instanceof Array)) onChange(name);
+      }}
+      getOptionDisabled={option =>
```

Comment:
should they be disabled? or should we just show nothing in the list? 
You can see that I went disabled.. you might not agree?

Reply:
I like it! but its also confusing what it means. Is it possible to give a hint like a green tick (or some text "already enrolled") to show that the patient is already enrolled in these programs (had this when first testing and was wondering why I can't select it)

Reply:
nice idea! thanks - was wondering about tooltip or something. will put something in.